nextflow/modules/contig_split_by_taxonomy/process/lca.nf

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env nextflow
+nextflow.enable.dsl=2
+
+process LCA {
+  label 'process_medium'
+  label 'process_long'
```

Comment:
I want to discuss the pickling. I don't remember if I already mentioned it but I don't think Autometa should be writing the pickle files to the same directory as the database (if I remember correctly, that is what it does). There should probably be a `storeDir` that it writes to, keeping tack of what version/date of a file it has pickled. This would allow us to also change the docker permissions to only `read`

This should be done in the download database step which is currently in another local branch of mine.